.github/workflows/inspektor-gadget.yml

```diff
@@ -1440,6 +1440,81 @@ jobs:
           test-step-conclusion: ${{ steps.gadgets-tests.conclusion }}
           test-summary-suffix: "gadgets-unittest"
 
+  gadgets-kernel-unittest:
+    name: Test gadgets on different kernel versions
```

Comment:
```suggestion
    name: Gadgets unit tests on kernel
```

This to avoid a long name that is trimmed on the Github UI

![image](https://github.com/user-attachments/assets/53d3bcac-1ec0-4e71-a328-ecb839c2841f)